scripts/cubecam.lua

```diff
@@ -182,6 +197,24 @@ function cubecam.on_value_changed(player, element)
   end
 end
 
+function cubecam.on_settings_changed(player, setting)
+  if setting == "cube-cubecam-scale" then
+    local state = player_state(player)
+    if not state.main then
+      return
+    end
+    local main = state.main
```

Comment:
I think it'd be simpler to just call
```
        cubecam_close(player)
        cubecam_open(player, player.fullscreen)
```
and it should apply the new size automatically. I don't think preserving the position matters that much since you won't be changing this often and it resets on open/close anyway.